src/meson.build

```diff
@@ -80,4 +80,4 @@ if get_option('tests')
         test_exe,
         timeout: 0
     )
-endif
\ No newline at end of file
+endif
```

Comment:
It is not clear on GitHub if you added or removed a trailing whitespace. Could you confirm ? After that I will merge, thanks!